### ISSUE TITLE
Drag slide gesture on mobile problem.

### ISSUE THREAD

**Issue by bushblade** [farbenmeer/react-spring-slider#161]
Hi I've recently implemented react-spring-slider in my own website and really like it.
I am having one issue though in that the gesture to slide to the next or previous image doesn't seem to work on a mobile device running chrome browser or in the dev tools responsive view of Chromium.
It works fine in FF dev tools and in FF on Android, it also doesn't work on safari on IOS.
I'm not sure if I'm doing/not doing something I should.
You can see the issue on [my website in any of the galleries](https://bushblade.co.uk/woodlore-clone/) 
Just open dev tools on Chrome/Chromium in responsive view or on real hardware mobile and try to swipe left/right.
I'll paste the portion of code from my component....
```js
      <Portal>
        <Modal open={isOpen}>
          <CloseModalButton onClick={() => setOpen(false)}>
            <span></span>
          </CloseModalButton>
          <Slider hasArrows hasBullets activeIndex={current}>
            {images.map((image) => (
              <SliderImageStyles key={image.key}>
                <img
                  src={image.src}
                  alt={image.originalName}
                  role="presentation"
                  onDragStart={(e) => {
                    e.preventDefault()
                    return false
                  }}
                />
              </SliderImageStyles>
            ))}
          </Slider>
        </Modal>
      </Portal>
```
I had to disable the default drag event of an image to get any drag to work at all. I did try using a background image on a div instead but had the same results, just it was harder to control the size of the images.
Any advice much appreciated.

**Comment by bushblade** [comment]
Also on FF I am seeing 
```
Will-change memory consumption is too high. Budget limit is the document surface area multiplied by 3 (1152718 px). Occurrences of will-change over the budget will be ignored
```
Whenever the slider is open.
The only place I have used `will-change` is in an unrelated part of the project, in the navbar.
I don't see the warning in Chromium.

**Comment by mismosmi** [comment]
Hi @bushblade , this appears to be working fine for me.
Is this still an issue?
Best,
Michel

https://user-images.githubusercontent.com/26325283/133423562-10df04a0-b3df-4a0f-9aef-a443183301e5.mov

**Comment by bushblade** [comment]
@mismosmi Hi, if you're checking my website today then it no longer uses react-spring-slider.
[I wrote and published my own component here.](https://www.npmjs.com/package/react-touch-drag-slider)
So that is what it uses now.

**Comment by bushblade** [comment]
I have no idea if the issue still persists as I switched to using my own library. Has this been fixed?

**Comment by mismosmi** [comment]
> I have no idea if the issue still persists as I switched to using my own library. I see it has now been closed but has this been fixed?

@bushblade yes, it has been fixed.

**Comment by Tobi-mmt** [comment]
Looks like the bot missed to comment this issue here 🤔
What the deploy wanted to say:

> :tada: This issue has been resolved in version 1.3.5 :tada:
>
>The release is available on:
>- [GitHub release](https://github.com/farbenmeer/react-spring-slider/releases/tag/v1.3.5)
>- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@farbenmeer/react-spring-slider/v/1.3.5)
>
>Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
